app/controllers/workflow/pages_controller.rb

```diff
@@ -26,17 +27,63 @@ def request_approval
         args = { f_uid: @cur_user._id, t_uid: workflow_approver[:user_id],
                  site: @cur_site, page: @item,
                  url: params[:url], comment: params[:workflow_comment] }
-        Workflow::Mailer.request_mail(args).deliver_now
+        Workflow::Mailer.request_mail(args).deliver_now rescue nil
       end
 
       @item.set_workflow_approver_state_to_request
       @item.update
     end
 
+    def workflow_alert
```

Comment:
__[rubocop]__
* Perceived complexity for workflow_alert is too high. [15/14]

<!-- Comment by [SideCI](https://sideci.com) -->